tests/all.js

```diff
@@ -1,17 +1,35 @@
-import http from "k6/http";
-
 import { allMethods } from "../methods/index.js";
 import { sendRpcRequest, assertSuccess } from "../utils/rpc.js";
 import { regularBenchmarkParams } from "../utils/benchmark_params.js";
+import { Trend, Rate } from 'k6/metrics';
 
 const url = __ENV.K6_TEST_URL || "http://localhost:2345/rpc/v1";
 
 export let options = regularBenchmarkParams;
 
+const methodTrends = {};
+const methodErrors = {};
+for (const method of allMethods) {
+  const key = safeMethodName(method.name);
+  if (!methodTrends[key]) {
+    methodTrends[key] = new Trend(key);
+  }
+  if (!methodErrors[key]) {
+    methodErrors[key] = new Rate(`_success_rate_${key}`);
+  }
+}
```

Comment:
Can you please add some comments around it? It's not clear from a quick glimpse what's the purpose of those collections.